implementing-nosto/implement-search/search/README.md

```diff
@@ -4,6 +4,8 @@
 
 For client-side/frontend integrations, Nosto's JavaScript library can be used to simplify the integration. It provides a programming interface to access the Search & Categories API.
 
+The functionality is available via the `nostojs` function in the browser window context when Nosto has been installed, or via the dedicated wrapper library [search-js](https://github.com/Nosto/search-js) for use in app builds.
```

Comment:
Consider,

>The functionality is available via the `nostojs` function in the browser window context when Nosto has been installed
The functionality is available via the `nostojs` function in the browser window context after Nosto has been loaded